sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -593,13 +593,24 @@ fromClause
     ;
 
 aggregationClause
-    : GROUP BY groupingExpressions+=expression (',' groupingExpressions+=expression)* (
+    : GROUP BY groupingExpressionWithGroupingAnalytics+=groupByClause
```

Comment:
The name `groupingExpressionWithGroupingAnalytics` is a bit long. The original name (`groupingExpressions`) looks fine to me. Any issue?

Reply:
> The name `groupingExpressionWithGroupingAnalytics` is a bit long. The original name (`groupingExpressions`) looks fine to me. Any issue?

Conflict with name in 
```
    | GROUP BY groupingExpressions+=expression (',' groupingExpressions+=expression)* (
      WITH kind=ROLLUP
    | WITH kind=CUBE
    | kind=GROUPING SETS '(' groupingSet (',' groupingSet)* ')')?
    ;
```
should. be different.

Reply:
Ah, ok, but `groupingExpressionWithGroupingAnalytics` -> `groupingExpressionsWithGroupingAnalytics `

Reply:
> Ah, ok, but `groupingExpressionWithGroupingAnalytics` -> `groupingExpressionsWithGroupingAnalytics `

Yea,